src/sql/azdata.proposed.d.ts

```diff
@@ -1338,5 +1342,24 @@ declare module 'azdata' {
 		 * Sets the ariaLabel for the right arrow button that shows up in clickable infoboxes
 		 */
 		clickableButtonAriaLabel?: string;
+
+		/**
+		 * Add links to infobox text.
```

Comment:
Please provide more detailed information about how these links are used (format of the text string and whatnot)